_includes/v19.2/sql/diagrams/unsplit.html

```diff
@@ -0,0 +1,25 @@
+<div><svg width="631" height="81">
```

Comment:
This file does not appear to be included from any pages.  Unless I'm missing it?